app/src/main/java/io/github/muntashirakon/setedit/Utils/Shortcut/ShortcutEditItemModel.java

```diff
@@ -0,0 +1,20 @@
+package io.github.muntashirakon.setedit.Utils.Shortcut;
+
+public class ShortcutEditItemModel {
+    private final String Name;
+    private final String Detail;
+
+    public ShortcutEditItemModel(String name, String detail) {
+        Name = name;
+        Detail = detail;
+    }
+
+    public String getSettingsType() {
```

Comment:
Maybe rename `Name` to `settingType`?

Reply:
Changed to shortcutContent more acurate.